internal/config/node/systemd.go

```diff
@@ -19,26 +17,13 @@ var (
 
 func SystemdHasCollectMode() bool {
 	systemdHasCollectModeOnce.Do(func() {
-		stdout, err := exec.Command("systemctl", "--version").Output()
+		// This will show whether the currently running systemd supports CollectMode
+		_, err := exec.Command("systemctl", "show", "-p", "CollectMode", "systemd").Output()
 		if err != nil {
-			systemdHasCollectModeErr = err
+			systemdHasCollectModeErr = errors.Wrapf(err, "Failed to check systemd CollectMode")
```

Comment:
nit, errors already indicate that something has failed and should start lower case:
```suggestion
			systemdHasCollectModeErr = errors.Wrapf(err, "check systemd CollectMode")
```

Reply:
good point, fixed!